src/vellum/workflows/nodes/experimental/tool_calling_node/utils.py

```diff
@@ -41,6 +42,15 @@ class Trigger(InlinePromptNode.Trigger):
         merge_behavior = MergeBehavior.AWAIT_ATTRIBUTES
 
     def run(self) -> Iterator[BaseOutput]:
+        if hasattr(self, "max_prompt_iterations") and self.max_prompt_iterations is not None:
```

Comment:
[nit] Let's default the attribute to None so that we don't need this `hasattr` check

Reply:
Actually, reading [Aaron's](https://www.notion.so/PRD-ToolCallingNode-1c39035c57bd8084b8a9defa1a767b31?d=1c39035c57bd807eab61001c6697f051&source=copy_link#1c39035c57bd8046a578d029cf5e5c0a) comment here, I'm seeing we should default to some concrete number, like 5.